AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md

```diff
@@ -23,7 +23,8 @@ You will learn how to train a CNN model with Keras and TensorFlow*, use Intel®
 |:---                               |:---         |
 | OS                                | Ubuntu* 20.04 (or newer) <br> Windows 11, 10* |
 | Hardware                          | Intel® Core™ Gen10 Processor <br> Intel® Xeon® Scalable Performance processors |
-| Software                          | Intel® Neural Compressor, Intel Optimization for TensorFlow |
+| Software                          | Intel® Neural Compressor, Intel® Extension for TensorFlow* |
+> **Note**: AI and Analytics samples are validated on AI Tools Offline Installer. For the full list of validated platforms refer to [Platform Validation](https://github.com/oneapi-src/oneAPI-samples/tree/master?tab=readme-ov-file#platform-validation).
 
 ### Intel® Neural Compressor and Sample Code Versions
 
```

Comment:
It seems like Intel® AI Tools has been renamed to just AI Tools, we need to update this everywhere in the README

Reply:
yes, update